### DISCUSSION TITLE
are key words case sensitive?

### DISCUSSION THREAD

**Discussion by aandyrea** [DocNow/twarc#669] [Q&A] [answered]
I am trying to collect German-language tweets with some key words. In German, nouns begin with a capital letter. However, I'm sure some people may tweet without capitalizing. 

My question is whether, in my code, I need to include, for example, both "immigrant" and Immigrant", or whether it's not case sensitive.

**Comment by igorbrigadir** [comment, marked as answer]
Nope! no case sensitivity in search APIs.

**Reply by aandyrea** [reply]
Thank you! (:
